CRM/MembershipExtras/SettingsManager.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Helps manage settings for the extension.
+ */
+class CRM_MembershipExtras_SettingsManager {
+
+  /**
+   * Returns the details of the default payment processor as per payment plan
+   * settings, or NULL if it does not exist.
+   *
+   * @return array
+   */
+  public static function getDefaultProcessor() {

```

Comment:
I think it is better to rename this to getDefaultProcessorId() and only return the ID, if we needed more info later then we can change it but for now lets just stick to the ID.